src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentsResource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentsResource : ResourceBase, ITrustlyPaymentsResource
+    {
+        public TrustlyPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+
+        public Task<TrustlyPayment> Create(TrustlyPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None)
+        {
+            return TrustlyPayment.Create(paymentRequest, this.httpClient, GetExpandQueryString(paymentExpand));
+        }
+
+
+        public Task<TrustlyPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)